### DISCUSSION TITLE
Contribute Cloudstack Project

### DISCUSSION THREAD

**Discussion by lohitslohit** [apache/cloudstack#9085] [Q&A] [answered]
Hi, I'm new to the CloudStack project and would like to contribute. How can I join the Slack community? I don't have an @apache.org email address. Also, how do I join the mailing list?

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by GutoVeronezi** [comment]
Hello, @lohitslohit 

Thanks for willing to contribute.

Most interactions occurs in the mailing list, PRs, issues, and discussions. For joining the Apache CloudStack mailing lists, you can take a look at this link: https://cloudstack.apache.org/mailing-lists/. For the official ASF Slack channels, you would need an @apache.org email or someone from inside to invite you.

**Comment by lohitslohit** [comment]
Sure Thanks for your update

**Comment by yadvr** [comment, marked as answer]
@lohitslohit to join CloudStack community slack  use this link - https://join.slack.com/t/apachecloudstack/shared_invite/zt-2j7e7nimo-1OUROPE9N79u1Fx7KW3vwQ (the ASF slack is limited by access). Slack tends to be less active, you're better off using the users@ ML (https://cloudstack.apache.org/mailing-lists) or project Github discussions/issues/pull requests.

To learn to contribute to CloudStack, you can start by picking any good first issues such as https://github.com/apache/cloudstack/issues?q=is%3Aopen+is%3Aissue+label%3Agood-first-issue and learn CloudStack development: https://github.com/shapeblue/hackerbook?tab=readme-ov-file#cloudstack-hackerbook
